plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -99,6 +101,14 @@ function MenuDialog:init()
     switch:setPosition(settings.enabled and 2 or 1)
     self:mergeLayoutInVertical(switch)
 
+    local edit_button = {
+        text = self.is_edit_mode and _("Leave edit mode") or _("Enter edit mode"),
```

Comment:
Or maybe something like edit mode/review mode?

To be clear, not a hint to change it (at this time).

Reply:
Since it's only one button, just an edit mode or review mode on it would be confusing. This is clearer without introducing another mode :)

Reply:
The crux of what I meant was trying to avoid the rather wordy enter/leave, whether with an icon or with words.

Reply:
Quick deletion/Resume ? Less wordy.

Reply:
Works for me :)

Reply:
Done.